src/WallnutCookiesDelivery.Web/Controllers/CartController.cs

```diff
@@ -0,0 +1,38 @@
+using System.Diagnostics;
+using System.Security.Claims;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using WallnutCookiesDelivery.Application.Interfaces.IServices;
+using WallnutCookiesDelivery.Web.ViewModels;
+
+namespace WallnutCookiesDelivery.Web.Controllers;
+
+public class CartController : Controller
+{
+    private readonly ICartService _cartService;
+    private readonly IMapper _mapper;
+    public CartController(ICartService cartService, IMapper mapper)
+    {
+        _cartService = cartService;
+        _mapper = mapper;
+    }
+
+    [HttpGet]
+    public IActionResult Index(CartViewModel cartModel)
+    {
+
+        return View(cartModel);
+    }
+
+    [HttpPost]
```

Comment:
you can try to add `[Authenticate]` attribute making sure that you have initialized and authenticated user